src/recommendations/ProductCard/index.jsx

```diff
@@ -75,11 +74,12 @@ const ProductCard = ({
     ],
   );
   const handleCardClick = () => {
-    trackRecommendationCardClickOptimizely(userId?.toString());
     trackRecommendationsClicked(
-      product.courseKey,
-      false,
+      product.uuid,
```

Comment:
does `uuid` contain `productKey` for all products? I think `uuid` is just for programs, what about courses, and other types?

Reply:
Currently Its is in all products but I write a function `generateProductKey` to make product key according to product.